CarpetCleaning/Views/dbo.JobsOverview

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW [JobSpecifics] AS
+    SELECT * FROM (
+    SELECT j.JobId, j.JobDate, t.[Name], a.AddressNumber, a.Street, a.Apt, c.City, 
+           a.State, a.Zipcode, j.Total,j.Discount, jd.Amount, st.ServiceName
+    from DBO.Jobs j
+    JOIN DBO.Technicians T ON t.TechnicianID = j.TechnicianID
+    JOIN DBO.Addresses A ON A.AddressID = J.AddressID
```

Comment:
Please make this SQL follow our established convention.  Change DBO to dbo, change A to a, like we have in the stored procedures.